README.md

```diff
@@ -363,14 +380,19 @@ browser warns the user.
 
 <img src="images/DuckDuckGo_Browser_Redirect_Warning.png" width=300px/>
 
-#### Save Browsers
+
+#### Safe Browsers
 The following browsers generate the intended 
 user experience:
 
 1. **Chrome** (especially Chrome Custom Tabs)
 2. **Firefox** (also supports Custom Tabs)
 3. **Opera** 
 
+JJH: Is this section only talking about custom url schemes?
```

Comment:
Yes because Android App Links are not supported by other browsers than Chrome anyway.

Reply:
I saw some hints that Firefox Fenix might support them, but it's unclear, I got lost in a maze of issues that started being about app links but seemed to pivot into talking about intent: scheme links.

Reply:
I just tested Firefox Fenix and it does not support Android App Links yet. Additionally they kind of broke the support for the intent:// scheme. If I use it in the Firefox Fenix, I am always getting redirected to the fallback URL. This is very bad!

Reply:
That sucks. It might be worth reporting a bug here: https://github.com/mozilla-mobile/fenix/issues ? not sure if we can give an easy to reproduce example that works in Chrome but not Fenix.

They disabled something related deliberately here, but I'm not sure exactly what: https://github.com/mozilla-mobile/fenix/pull/3359/files and if there was any further changes.

Reply:
I just made an interesting discovery: In the new Fenix browser there is an option ``Open links in apps`` under ``Settings | Advanced``. This option is disabled by default but after enabling it not only the intent:// scheme works also Android App Links are supported. Maybe this is related to the PR you sent me. But then I wonder why they intentionally disable this feature because it would improve the UX a lot and also the security if Android App Links work.

Reply:
Ah, well spotted. I don't think it's doing app links - at least I couldn't find any code that seemed like it covered it - the relevant code seems to be here: https://github.com/mozilla-mobile/android-components/tree/master/components/feature/app-links/src/main/java/mozilla/components/feature/app/links

I think there are a number of vocal people in the Firefox user community that don't like links opening in apps :-( I also saw references to the fact that the developers want to add a "are you okay to open the app" type dialog before enabling it by default. (They apparently already have a dialog like that when in 'private browsing' mode.)

Reply:
I just tested the handling of Android App Links in Fenix and it does not seem to handle them correctly. I think they just search for an app that claims the URL (but not necessarily verify its relationship with the domain) and if only one app that claims the URL is installed it opens up without further user interaction. So this is definitively a problem because every app can claim the URL without verifying it. 

But I guess the bigger problem here is that the option ``Open links in apps`` is disabled by default and I think the average user will not enable it because they don't even know that this option exists. So URLs with the intent:// scheme will also not work for these users.